README.md

```diff
@@ -105,36 +125,62 @@ themselves from scratch, meaning they must know (or at least guess) the model.
 PPL users must also know how to specify such a model in a way that is
 compatible with the underlying inference procedure.
 
-### Who should not use lace
+### Example use cases
+
+- **Combine data sources and understand how they interact.** For example, we
+    may wish to predict cognitive decline from demographics, survey or task
+    performance, EKG data, and other clinical data. Combined, this data would
+    typically be very sparse (most patients will not have all fields filled
+    in), and it is difficult to know how to explicitly model the interaction of
+    these data layers. In Lace, we would just concatenate the layers and run
```

Comment:
"Concatenate the layers and pull them through"-i don't understand this, it sounds too simple-like how could it be that easy? Maybe a rewording that explains why you can't do that yourself? Or maybe I'm not getting it

Reply:
It is that easy 😉